Examples/Framework/include/ActsExamples/Validation/TrackClassification.hpp

```diff
@@ -57,4 +57,9 @@ void identifyContributingParticles(
     const Trajectories& trajectories, size_t trajectoryTip,
     std::vector<ParticleHitCount>& particleHitCounts);
 
+void identifyContributingParticles(
```

Comment:
A bit unrelated, but since we are touching this function... I always disliked the interface with passing a reference to a vector here. This seems an unnecessary optimization that makes the code less clear.

If there are no objections, I would prefer to return a `boost::small_vector<ActsFatras::Barcode, 5> or something like this, this should be sufficient storage in most cases...

Reply:
right that seems a bit odd. I guess another alternative would be to only look for the majority particle as this is the one we care about at the end anyways I think?

but I would keep that rather outside of this PR as it is kinda big already